ui/admin/app/controllers/application.js

```diff
@@ -39,6 +40,13 @@ export default class ApplicationController extends Controller {
     return THEMES;
   }
 
+  get showSideNav() {
+    return (
+      this.router.currentRouteName.startsWith('scopes.scope') &&
```

Comment:
Nice work. 

Just curious, I noticed your PR description mentions "The side nav should not be visible on the onboarding, change-password , and authenticate routes."

Can you share a bit of context around why the side nav should only appear on routes that start with `scopes.scope`?

Reply:
Great question, this is to keep the functionality the same as before. Additionally, `scopes.scope` is the route where we load global scope and org scopes (& project scopes if in project level) and having those scopes loaded is necessary for displaying sub routes and the scope picker.